src/msi/Repl.cs

```diff
@@ -432,6 +432,15 @@ private void EvaluateMetaCommand(string input)
             var inQuotes = false;
             var position = 1;
             var sb = new StringBuilder();
+
+            void CommitPendingArgument()
```

Comment:
Why did this function move? Seems unrelated.

Reply:
I was checking if the order of declaration matters... I forgot to discard the change after I've checked it. Sorry about that!

Reply:
Fixed it!